wagtail/images/tests/test_admin_views.py

```diff
@@ -1787,6 +1787,10 @@ def test_select_format_flag_after_upload_form_error_bad_extension(self):
             },
         )
 
+        SUPPORTED_FORMATS_TEXT = ", ".join(
+            ["gif", "jpg", "jpeg", "png", "webp"]
+        ).upper()
+
```

Comment:
This feels like including implementation detail in the tests. Since the string won't change anyway, maybe it's better to just hardcode it?